BotNet.Services/StatusPage/StatusPageClient.cs

```diff
@@ -49,6 +49,14 @@ public sealed class StatusPageClient(HttpClient httpClient) {
 			{ "Slack", "https://status.slack.com/api/v2.0.0/current" }
 		};
 
+		// Meta services - use HTTP availability check (no public status API)
+		private static readonly Dictionary<string, string> HttpAvailabilityServices = new() {
+			{ "Facebook", "https://www.facebook.com" },
+			{ "Instagram", "https://www.instagram.com" },
+			{ "WhatsApp", "https://www.whatsapp.com" },
+			{ "Threads", "https://www.threads.net" }
```

Comment:
I don't think checking the status of landing page of a mobile app constitutes checking the status of the service itself. Is there any health check endpoint or any API endpoints you can poll? Also, check various parts of the service independently